src/main/java/frc/robot/subsystems/superstructure/arm/ArmIO.java

```diff
@@ -18,6 +16,9 @@ class ArmIOInputs {
         double rightPivotVoltageVolts = 0;
         double rightPivotTorqueCurrentAmps = 0;
         double rightPivotTempCelsius = 0;
+
+        double pivotEncoderPositionRots = 0;
```

Comment:
No need to log this, we already have the position from both motors - see comment about fusing encoder data.

Reply:
Actually, not sure if we should log this anyways, in case the 2 positions deviate. Any thoughts?

Reply:
I say just keep it. No harm in having it